### PR TITLE
api is not exiting on unknown Event type

### LINKED ISSUE #1017: Not implemented Event type on chain breaks api
- [x] I have searched the [issues](https://github.com/openkfw/TruBudget/issues) of this repository and believe that this is not a duplicate.

## Description 😯

If Trubudget's api reads an event type which is not implemented it exits with error code 1. I agree that this should be an error/warning, but it should not lead to an program exit.

### How to reproduce 🕹

Store an blockchain event with an unknown event type and start the api

## Your Environment 🌎

| Tech            | Version |
| --------------- | ------- |
| API             | v1.27.0  |
| Blockchain      | v1.27.0  |

### PR DESCRIPTION
**Pull request by Stezido**
### Checklist

- [x] I have followed (at least) the [PR section of the contributing guide](https://github.com/openkfw/TruBudget/blob/master/CONTRIBUTING.md).
- [ ] I fixed all necessary PR warnings
- [x] The commit history is clean
- [ ] The E2E tests are passing
- [ ] If possible, the issue has been divided into more subtasks
- [x] I did a self review before requesting a review from another team member

### Description
This PR is only a workarround. The main issue is that documents stored via mutlichains offchain mechanic are stored in trubudget's cache. In a future fix these documents should not be part of the cache anymore.
Tasks done in this PR:
- Skip and log errors happening while converting to readable items
- Adapt error messages including all necessary information
- Eventsourcing of workflowitems skip the workflowitem_document_uploaded event since document uploads are handled via workflowitem_update event

### Additional Change

It was not possible to exclude certain vulnerabilities to pass the audit checks if there is no fix for the vulnerability yet.
With the new package better-npm-audit you can exclude vulnerabilities with a command argument or with a file called `.nsprc`.

Closes #1017

### CODE REVIEW

File: .github/workflows/github-ci.yml [hunk 1]
```diff
@@ -45,17 +45,16 @@ jobs:
     strategy:
       matrix:
         node-version: [10.x]
-        project:
-          [
+        project: [
             frontend,
             api,
-            blockchain,
+            # blockchain,
```
Comment (mayrmartin) [inline .github/workflows/github-ci.yml:51]: Consider removing if not used

File: frontend/docker-compose.yml [hunk 2]
```diff
@@ -1,37 +1,43 @@
 version: "3.2"
 services:
   master:
-    image: trubudget/blockchain:latest
+    image: trubudget/blockchain:v1.27.0
```
Comment (mayrmartin) [inline frontend/docker-compose.yml:4]: Why lock the TB version to 1.27? Also, why are the ports set to a fixed value & why is provisioning disabled in the docker-compose.yml?

Comment (Stezido) [reply frontend/docker-compose.yml:4]: sorry didn't want to push this file is was for testing purposes

File: .github/workflows/github-ci.yml (hunk 1 above)
Comment (Stezido) [reply .github/workflows/github-ci.yml:51]: currently the blockchain project has some moderate vulnerabilities so the pipeline wouldn't go through.

**Review by georgimld — APPROVED** [review]

**Pull request merged**